packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/tests/index.test.js

```diff
@@ -433,4 +440,38 @@ describe('BlocksEditor toolbar', () => {
     const blocksDropdown = screen.getByRole('combobox', { name: /Select a block/i });
     expect(within(blocksDropdown).getByText(/heading 1/i)).toBeInTheDocument();
   });
+
+  it('creates a new code block without empty lines before it when you select the option in a empty editor', async () => {
+    setup(emptyInitialValue);
+
+    // Convert selection to a code block
+    const blocksDropdown = screen.getByRole('combobox', { name: /Select a block/i });
+    await user.click(blocksDropdown);
+    await user.click(screen.getByRole('option', { name: 'Code' }));
+
+    expect(baseEditor.children).toEqual([
+      {
+        type: 'code',
+        format: null,
+        level: null,
```

Comment:
Level doesn't belong here

Reply:
It is coming from there https://github.com/strapi/strapi/blob/abaa057cf3e2740ade467baf96059e4617bc87da/packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js#L147 we need to define the format and level props because, otherwise when you insert a Node https://github.com/strapi/strapi/blob/abaa057cf3e2740ade467baf96059e4617bc87da/packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js#L172 for example when you try to convert a h1 to a paragraph without editor selection the level is passed in the children of the new paragraph